oai-pmh-handler/src/main/java/no/sikt/nva/oai/pmh/handler/oaipmh/transformers/XmlUtils.java

```diff
@@ -0,0 +1,12 @@
+package no.sikt.nva.oai.pmh.handler.oaipmh.transformers;
+
+import static java.util.Objects.isNull;
+
+public final class XmlUtils {
+
+  private XmlUtils() {}
+
+  public static String sanitizeXmlValue(String value) {
+    return isNull(value) ? null : value.replaceAll("[\\x00-\\x08\\x0B\\x0C\\x0E-\\x1F]", "").trim();
```

Comment:
We should consider if we want to remove this stuff elsewhere in NVA (since we have other XML workflows), please create a task